packages/react-sdk/src/components/image.props.json

```diff
@@ -2538,5 +2538,41 @@
     "control": {
       "type": "number"
     }
+  },
+  "loader": {
+    "defaultValue": null,
+    "description": "",
+    "name": "loader",
+    "declarations": [
+      {
+        "fileName": "src/components/image.tsx",
+        "name": "TypeLiteral"
+      }
+    ],
+    "required": false,
+    "type": {
+      "name": "ImageLoader"
+    },
+    "control": {
+      "type": "text"
+    }
+  },
```

Comment:
Just something to keep in mind, not necessarily an issue depending on how we going to use this prop: 

We keep props in DB as a JSON object, and it's impossible to store a function there.